apps/etterlatte-beregning/src/main/kotlin/beregning/BeregningService.kt

```diff
@@ -108,6 +124,12 @@ class BeregningService(
         this?.copy(overstyrBeregning = hentOverstyrBeregningPaaBehandlingId(behandlingId, brukerTokenInfo))
 }
 
+class KanIkkeDeaktivereOverstyrtBeregningForBehandlingPaaSak :
+    UgyldigForespoerselException(
+        code = "UGYLDIG_STATUS_BEHANDLING",
+        detail = "Behandlingen kan ikke endre overstyrt beregning da den har feil status",
```

Comment:
Kanskje legge inn "eller ikke er førstegangsbehandling". Evnt egen exception for den casen.